checkOpenPMD_h5.py

```diff
@@ -807,15 +831,21 @@ def check_particles(f, iteration, v, pic) :
 
     # root attributes at "/"
     result_array = np.array([0, 0])
-    result_array += check_root_attr(f, verbose, extension_pic)
+    result_array += check_root_attr(f, verbose)
+    
+    extensionStates = get_extensions(f, verbose)
+    if extension_pic and not extensionStates["ED-PIC"] :
```

Comment:
can you please rename the variable `extension_pic` to `force_extension_pic` to make it self-documented? :)